code/datums/components/food/decomposition.dm

```diff
@@ -24,7 +25,7 @@
 	var/produce_ants = FALSE
 
 /datum/component/decomposition/Initialize(mapload, decomp_req_handle, decomp_flags = NONE, decomp_result, ant_attracting = FALSE, custom_time = 0)
-	if(!isobj(parent))
+	if(!isobj(parent) || !parent.GetComponent(/datum/component/germ_sensitive))
```

Comment:
should use a trait, something like TRAIT_GERM_SENSITIVE, to avoid the GetComponent call